k8s/mongodb/mongo-ss.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 768Mi
+            memory: 3.5G
```

Comment:
Suggestion 2 from @krish7919 sounds reasonable. I've never used k8s scheduling constraints but if they can in fact keep Mongo by itself on an agent, that would be cool.